profiles/anekemichael.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/62024664/183684520-33c4059b-d4cc-4404-bfc9-9407370944e3.png",
+  "issueId": 66,
```

Comment:
This should be the correct issue ID: #1309.
```suggestion
  "issueId": 1309,
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable